src/Firehose.Web/Infrastructure/NewCombinedFeedSource.cs

```diff
@@ -182,6 +182,27 @@ DateTimeOffset GetMaxTime(SyndicationItem item)
 				LastUpdatedTime = DateTimeOffset.UtcNow
             };
 
+            // This foreach section is responsible for adding the Twitter handles
+            foreach (var item in feed.Items)
+            {
+                string newTitleText = string.Empty;
+                if (item.Authors.Count > 0)
+                    newTitleText = Tamarins.Where(x => x.FirstName + " " + x.LastName == item.Authors[0].Name).FirstOrDefault()?.TwitterHandle ?? string.Empty;
```

Comment:
This matching code seems to be a bit fragile and could perhaps lead to many misses depending on what the author has set up their feed to return.

Reply:
Yup, @Cheesebaron, I couldn't find a different way to find a match except by comparing with `item.Authors[0].Name`. Do you have any other ideas?

Reply:
Significantly changed this, and just ended up using the function used by the Preview page